code/game/machinery/computer/computer.dm

```diff
@@ -68,24 +68,24 @@
 	update_icon()
 	flickering = FALSE
 
-/obj/machinery/computer/update_icon()
-	cut_overlays()
+/obj/machinery/computer/update_overlays()
+	. = ..()
 	underlays.Cut()
```

Comment:
Ideally, but that started a different rabbit hole when I went through usage of underlays in the files so that'd just increase the number of modified files further. Also, that's not what underlays are. In terms of general attributes yes, but they're drawn under the icon, which is a behavior I'm deliberately using to avoid a quirk of Byond right-clicking.